core/src/rpc/aggregator.rs

```
@@ -1359,11 +1339,14 @@ impl ClementineAggregator for Aggregator {
         .await?;
 
         let signed_movetx_handler = self
-            .send_movetx(move_to_vault_sigs, movetx_agg_nonce, deposit_params)
+            .create_movetx(move_to_vault_sigs, movetx_agg_nonce, deposit_params)
             .await?;
-        let txid = *signed_movetx_handler.get_txid();
 
-        Ok(Response::new(txid.into()))
+        let raw_signed_tx = RawSignedTx {
+            raw_tx: bitcoin::consensus::serialize(&signed_movetx_handler.get_cached_tx()),
+        };
+
+        Ok(Response::new(raw_signed_tx))
```

I think it would be better to send movetx here if automation is enabled for conveniency, but returning movetx instead of txid is good in case its not enabled.

---

I think it's better to have a consistent interface like this. In this way, if we incorrectly compile the aggregator, we'll get a clear error mesage. Otherwise, we would get a silent failure which we'd have to debug (i.e. we'd be debugging why movetx wasn't being sent)